content/1.7/back-office/dashboard/dashboard.md

```diff
@@ -0,0 +1,179 @@
+
+## Columns
+
+By default there are 8 fields in the dashboard:
+There are 2 columns in the dashboard: 
+
+- the left column: this column presents the data from the Dashboard Activity module
+- the central column: there are three blocks of content in this column, which are handled by modules
+- the right column: this column presents a "PrestaShop upgrade" block 
+
+## Central column
+
+### Dashboard (Dashboard Trends module)
+
+## Call to action
+
+On the top right of this field there are 2 buttons:
+
+ - **Configure** - opens a new page where various expenses of the store can be configured, in order to better indicate the trends. [dashboard-configure]({{<ref "dashboard-configure.md">}} "dashboard configure")
+ - **Refresh** - refreshes the stats of the shop.
+
+#### Tabs
+
+The following tabs are displayed and can be selected to display the evolution.
+
+ - **Sales** - displays the sales amount in the default currency tax excluded within the date range. A tooltip is displayed: "Sum of revenue (excl. tax) generated within the range by orders considered validated.".
+ - **Orders** - displays the number of orders within the date range. A tooltip is displayed: "Total number of orders received within the date range that are considered validated.".
+ - **Cart Value** - displays the average cart value in the default currency tax excluded within the date range. A tooltip is displayed: "Average Cart Value is a metric representing the value of an average order within the date range. It is calculated by dividing Sales by Orders.".
+ - **Visits** - displays the number of visits within the date range. A tooltip is displayed: "Total number of visits within the date range. A visit is the period of time a user is actively engaged with your website.".
+ - **Conversion Rate** - displays the conversion rate in percentage. A tooltip is displayed: "Ecommerce Conversion Rate is the percentage of visits that resulted in an validated order.".
+ - **Net Profit** - displays the net profit in the default currency tax excluded. A tooltip is displayed: "Net profit is a measure of the profitability of a venture after accounting for all Ecommerce costs. You can provide these costs by clicking on the configuration icon right above here.".
+
+After selecting a tab, the corresponding graphic is displayed. When hovering the graphic with the mouse, the details are displayed.
+Additionally, when comparing two time periods, it displays the evolution of each figure in percentages.
+
+## Left column
+
+### Activity overview (Dashboard Activity module)
+
+#### Call to action 
+
+On the top right of this block there are 2 buttons:
+
+ - **Configure** - when pressed, changes the activity overview to a configuration block with the following field:
+    - **Active cart** - drop down field with options: 15, 30, 45, 60, 90, 120.
+    - **Online visitors** - drop down field with options: 15, 30, 45, 60, 90, 120.
+    - **Abandoned cart (min)** - input field measured by "d".
```

Comment:
I don't understand what do you mean by "measured by "d"" ?
Could you specify the default value and the tooltip ?